### PR TITLE
fix package name in ci workflow

### PR DESCRIPTION
**Pull request by flynneva**
Signed-off-by: Evan Flynn <evan.flynn@apex.ai>

### CODE REVIEW

**Comment by flynneva** [comment]
@LanderU the ci should now run for all active ROS2 versions, apologies for all the pings/notifications 😓  its erroring now because we dont have any code in this repo until we merge over from `ApexAI/reference-system-autoware`

**Comment by flynneva** [comment]
for reference im just going off what I setup for the [`usb_cam` package](https://github.com/ros-drivers/usb_cam/tree/ros2)

File: .github/workflows/colcon-build.yml [hunk 1]
```diff
@@ -18,14 +18,37 @@ jobs:
...
+          # Rolling Ridley  (June 2020 - Present)
+          - ros_distribution: rolling
+            docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-rolling-ros-base-latest
+    container:
+      image: ${{ matrix.docker_image }}
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - uses: ros-tooling/setup-ros@v0.2
+      - name: setup workspace
+        run: mkdir -p ros_ws/src
```
Comment (LanderU) [inline .github/workflows/colcon-build.yml:44]:
```suggestion
        run: mkdir -p ros2_ws/src
```

File: .github/workflows/colcon-build.yml [hunk 2]
```diff
@@ -18,14 +18,37 @@ jobs:
...
+      - name: checkout
+        uses: actions/checkout@v2
         with:
-          required-ros-distributions: galactic
-      - uses: ros-tooling/action-ros-ci@v0.2
+          path: ros_ws/src
```
Comment (LanderU) [inline .github/workflows/colcon-build.yml:48]:
```suggestion
          path: ros2_ws/src
```

**Comment by LanderU** [comment]
@flynneva, no problem with the pings :), do you have an idea when do you include a package here? We can merge this and then adjust when the package/s comes.

**Comment by flynneva** [comment]
I had the same question - maybe after the two open PRs get merged in then we can bring over the repo from the ApexAI fork

**Comment by flynneva** [comment]
@LanderU do you think you could merge this in so we could test #9 properly?

File: .github/workflows/colcon-build.yml [hunk 3]
```diff
@@ -18,14 +18,37 @@ jobs:
...
         with:
-          required-ros-distributions: galactic
-      - uses: ros-tooling/action-ros-ci@v0.2
+          path: ros2_ws/src
+      - name: build and test
+        uses: ros-tooling/action-ros-ci@v0.2
         with:
-          package-name: my_package
-          target-ros2-distro: galactic
+          package-name: reference_system_autoware
```
Comment (LanderU) [inline .github/workflows/colcon-build.yml:52]:
```suggestion
          package-name: reference_system
```

**Comment by LanderU** [comment]
@flynneva, if you apply the change I'll merge this.

**Review by LanderU — APPROVED** [review]
LGTM!

**Pull request merged**